code/processes/wdb.q

```diff
@@ -180,26 +182,26 @@ savetodisk:{[] savetables[savedir;getpartition[];0b;] each tablelist[]};
 
 /- eod - flush remaining data to disk
 endofday:{[pt]
-	.lg.o[`eod;"end of day message received - ",spt:string pt];	
-	/- create a dictionary of tables and merge limits
-	mergelimits:(tablelist[],())!({[x] mergenumrows^mergemaxrows[x]}tablelist[]),();	
-  tablist:tablelist[]!{0#value x} each tablelist[];
-	/ - if save mode is enabled then flush all data to disk
-	if[saveenabled;
-		endofdaysave[savedir;pt];
-		/ - if sort mode enable call endofdaysort within the process,else inform the sort and reload process to do it
-		$[sortenabled;endofdaysort;informsortandreload] . (savedir;pt;tablist;writedownmode;mergelimits;hdbsettings)];
-	.lg.o[`eod;"deleting data from tabsizes"];
-	@[`.wdb;`tabsizes;0#];
-	.lg.o[`eod;"end of day is now complete"];
-	};
+        .lg.o[`eod;"end of day message received - ",spt:string pt];	
```

Comment:
I would use 4 spaces instead of 8 here, the other indentation in the file is mostly tabs.